main.nf

```diff
@@ -244,16 +244,16 @@ process picard {
     module 'picard/2.0.1'
     module 'BEDTools'
     module 'samtools'
-    
+
     cpus 2
     memory { 16.GB * task.attempt }
     time { 24.h * task.attempt }
     errorStrategy { task.exitStatus == 143 ? 'retry' : 'terminate' }
     maxRetries 3
     maxErrors '-1'
-    
+
     publishDir "${params.outdir}/picard", mode: 'copy'
-    
+
     input:
     file bam_picard
     set val(prefix) from name_for_picard
```

Comment:
See the [other comment below](https://github.com/SciLifeLab/NGI-ChIPseq/pull/4/files/e836f3f74c85ac3dd83cf378873ed5deab42205d#r67351361)